scripts/_common.sh

```diff
@@ -11,6 +11,26 @@ pkg_dependencies="postgresql postgresql-contrib"
 # PERSONAL HELPERS
 #=================================================
 
+# custom function to detect armv6 and armv7
+# ($YNH_ARCH returns armhf for both...)
+detect_arch(){
+	local architecture
+	if [ -n "$(uname -m | grep arm64)" ] || [ -n "$(uname -m | grep aarch64)" ]; then
+		architecture="arm64"
+	elif [ -n "$(uname -m | grep 64)" ]; then
+		architecture="x86-64"
+	elif [ -n "$(uname -m | grep 86)" ]; then
+		architecture="i386"
+	elif [ -n "$(uname -m | grep armv6)" ]; then
+		architecture="armv6"
+	elif [ -n "$(uname -m | grep armv7)" ]; then
+		architecture="armv7"
```

Comment:
```suggestion
	elif [ -n "$(uname -m | grep armv6)" ]; then
		architecture="armv6l"
	elif [ -n "$(uname -m | grep armv7)" ]; then
		architecture="armv7l"
```
It seems you renamed the `.src` files.

Reply:
And thanks for the help ☺️